server.R

```diff
@@ -53,17 +54,35 @@ shinyServer(function(input, output, session) {
       hot_table(highlightCol = TRUE, highlightRow = TRUE)
   })    
   
+  # Download Parameter template
+  output$download_template <- downloadHandler(
+    filename = function() {
+      paste("Reform_", 
+      # Below is extracting the Tax year for the file names of reform parameter file.
+            basename(input$parameters_SQ) |>
+              strsplit("[_.]") |>
+              sapply(function(x) x[2]), 
+            ".xlsx", sep = "")
```

Comment:
These pipes are from R version 4 I think, and so when I tested this locally with R 3.6.2 (note, same major version as IDI) it errored out. Can you either change them to magrittr pipes "%>%", or maybe there is another way to make it compatible with R 3.6.2.

Reply:
Sure, I can do that, I was just testing it, since I am using R version 4